### PR TITLE
Remove missed `make` targets and GitHub Action pipelines for Schematron develop

### PR DESCRIPTION
**Pull request by aj-stein-gsa**
# Committer Notes

I missed this code and build support tooling. This is a small step towards completing #738.

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/GSA/fedramp-automation/blob/master/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/GSA/fedramp-automation/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] If applicable, does this PR reference the issue it addresses and explain how it addresses the issue?
- ~If applicable, have all [FedRAMP Documents Related to OSCAL Adoption](https://github.com/GSA/fedramp-automation) affected by the changes in this issue have been updated?~ N/A for this issue, it for team-focused development tooling.

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

### CODE REVIEW

**Review by Gabeblis — DISMISSED** [review]

**Comment by wandmagic** [comment]
We still use make test, and i think we should use make build for the content stuff, we just need to remove the build-web part

File: Makefile
Comment (wandmagic) [inline Makefile:1]: build and test should look like this:

test: build-validations ## Test all

build: init-content test-content build-content  ## Build all artifacts and copy into dist directory

**Review by wandmagic — CHANGES_REQUESTED** [review]

File: Makefile
Comment (aj-stein-gsa) [reply Makefile:1]: Sorry serves me write I was too aggressive yesterday with quick edits.

Comment (aj-stein-gsa) [reply Makefile:1]: Fixed commit on rebase and pushed, thanks for catching, you know, the one target I could not change. 😆

File: Makefile [hunk 1]
```diff
@@ -29,22 +29,10 @@ clean-oci-image:
 	docker rmi -f \
 		validation-tools:$(OCI_REV_TAG) \
 		ghcr.io/gsa/fedramp-automation/validation-tools:$(OCI_REV_TAG) \
-		gsatts/validation-tools:$(OCI_REV_TAG) \
+		gsatts/validation-tools:$(OCI_REV_TAG)
 
 test: build-validations ## Test all
```
Comment (wandmagic) [inline Makefile:35]:
```suggestion
build: init-content test-content build-content  ## Build all artifacts and copy into dist directory
```

Comment (aj-stein-gsa) [reply Makefile:35]: Sigh I am embarassed I waste this much time on this. I actually run these all separately and didn't know we use `build`, all those sub-targets are defined in the other module and I have never seen someone mention using these in chat, but you did now saw it three times, so there's that!

**Review by wandmagic — APPROVED** [review]

**Review by Gabeblis — APPROVED** [review]

**Pull request merged**